README.md

```diff
@@ -200,6 +204,8 @@ directory.
 
 positional arguments:
   <test-name>           Name of the test folder
+                        Add `ips` to the test name if you want it to run in IPS mode
+                        the test is run.
```

Comment:
nits:
1. I'm realizing this README output is quite mature than the actual help output of the script. Maybe we should make it consistent. wdyt?
2. Help outputs are generally supposed to be one liners w minimal info. Should such additional info stay in the docs?

Reply:
> nits:
> 
>     1. I'm realizing this README output is quite mature than the actual help output of the script. Maybe we should make it consistent. wdyt?
> 

Agreed!

>     2. Help outputs are generally supposed to be one liners w minimal info. Should such additional info stay in the docs?

I still think that this info should be outputted when we have the help, as I can imagine people would often first refer to the `help` output before going to the documentation. I can try to make it more concise, somehow, then.